internal/host/static/repository_host.go

```diff
@@ -133,7 +133,7 @@ func (r *Repository) UpdateHost(ctx context.Context, scopeId string, h *Host, ve
 				return nil, db.NoRowsAffected, errors.New(errors.InvalidAddress, op, "invalid address")
 			}
 		default:
-			return nil, db.NoRowsAffected, errors.New(errors.InvalidFieldMask, op, fmt.Sprintf("field: %s", f))
+			return nil, db.NoRowsAffected, errors.New(errors.InvalidFieldMask, op, fmt.Sprintf("invalid field mask: %s", f))
```

Comment:
What will the final error string for this be?

Reply:
So I made this change because the default message gets dropped when a custom message is supplied so this change makes it go from:
```static.UpdateHost: field: Bilbo: parameter violation: error #103`
to
```static.UpdateHost: invalid field mask: Bilbo: parameter violation: error #103```

Reply:
@mgaffney I felt having the text `invalid field mask` in the string rather than needing to rely on the `#103` was useful